contracts/BatchExchange.sol

```diff
@@ -381,8 +381,8 @@ contract BatchExchange is EpochTokenLocker {
       * @param user address of user whose orders are being queried
       * @return encoded bytes representing all orders
       */
-    function getEncodedUserOrders(address user) public view returns (bytes memory elements) {
-        for (uint256 i = 0; i < orders[user].length; i++) {
+    function getEncodedUserOrders(address user, uint256 offset, uint16 pageSize) public view returns (bytes memory elements) {
```

Comment:
`offset + pagesize` is an `orderIndex` so shouldn't `offset` also be a `uint16`?